src/test/resources/data/incorporation/LP5D_delta.json

```diff
@@ -0,0 +1,18 @@
+{
+  "filing_history": [
+    {
+      "category": "6",
+      "receive_date": "20220803113127",
+      "form_type": "LP5D",
+      "description": "REGISTRATION OF LIMITED PARTNERSHIP",
```

Comment:
in the transform rules the description is registration-as-a-limited-partnership (as a), does it make any difference?